projects/packages/sync/src/class-actions.php

```diff
@@ -855,6 +856,21 @@ public static function add_woocommerce_hpos_order_sync_module( $sync_modules ) {
 		return $sync_modules;
 	}
 
+	/**
+	 * Adds Woo's Product Meta Lookup sync module to existing modules for sending.
+	 *
+	 * @param array $sync_modules The list of sync modules declared prior to this filter.
+	 *
+	 * @access public
+	 * @static
+	 *
+	 * @return array A list of sync modules that now includes Woo's Product Meta Lookup module.
+	 */
+	public static function add_woocommerce_product_meta_lookup_sync_module( $sync_modules ) {
+		$sync_modules[] = 'Automattic\\Jetpack\\Sync\\Modules\\WooCommerce_Product_Meta_Lookup';
+		return $sync_modules;
+	}
```

Comment:
> I also echo what @darssen mentioned, that do we want to enable this by default for every Woo site? or should we keep it disabled for now, maybe allowing it to be enabled by filter ( so we can enable it on the Woo Analytics end )?
I did lean towards being conservative for now, and optionally enabling it.

That sounds good.  We could revert the changes to this file so it's not part of sync_modules by default, and then have WooCommerce Analytics enable it as needed using the `jetpack_sync_modules` filter.